cachi2/core/package_managers/pip.py

```diff
@@ -1841,14 +1868,13 @@ def _process_package_distributions(
                     "Alternately, allow the use of wheels."
                 )
                 docs = PIP_NO_SDIST_DOC
-
             raise PackageRejected(
                 f"No distributions found for package {name}=={version}",
                 solution=solution,
                 docs=docs,
             )
 
-    return best_sdist, wheels
+    return best_sdist + wheels
```

Comment:
I think `[best_sdist] + wheels` is perfectly acceptable here.

Reply:
How about this one?